examples/plugins/train-local-rot13/lib/train-local-rot13/transport.rb

```diff
@@ -0,0 +1,27 @@
+
+# Train Plugins v1 are usually declared under the TrainPlugins namespace.
+# Each plugin has three components: Transport, Connection, and Platform.
+# We'll only define the Transport here, but we'll refer to the others.
+# require 'train-local-rot13/connection'
```

Comment:
Can we remove this `require`?

Reply:
Not sure how it was working without it.  uncommenting.